components/dashboard/src/teams/TeamUsage.tsx

```diff
@@ -211,60 +217,93 @@ function TeamUsage() {
                         )}
                         {billedUsage.length > 0 && !isLoading && (
                             <div className="flex flex-col w-full mb-8">
-                                <ItemsList className="mt-2 text-gray-500">
-                                    <Item header={false} className="grid grid-cols-5 bg-gray-100 mb-5">
-                                        <ItemField className="my-auto">
+                                <ItemsList className="mt-2 text-gray-400 dark:text-gray-500">
+                                    <Item header={false} className="grid grid-cols-12 gap-x-3 bg-gray-100 mb-5">
+                                        <ItemField className="col-span-2 my-auto ">
                                             <span>Type</span>
                                         </ItemField>
-                                        <ItemField className="my-auto">
-                                            <span>Class</span>
+                                        <ItemField className="col-span-5 my-auto">
+                                            <span>ID</span>
                                         </ItemField>
                                         <ItemField className="my-auto">
-                                            <span>Usage</span>
-                                        </ItemField>
-                                        <ItemField className="flex my-auto">
-                                            <CreditsSvg className="my-auto mr-1" />
                                             <span>Credits</span>
                                         </ItemField>
-                                        <ItemField className="my-auto cursor-pointer">
+                                        <ItemField className="my-auto" />
+                                        <ItemField className="col-span-3 my-auto cursor-pointer">
                                             <span
                                                 className="flex my-auto"
                                                 onClick={() => setIsStartedTimeDescending(!isStartedTimeDescending)}
                                             >
-                                                Started
+                                                Timestamp
                                                 <SortArrow
-                                                    className={`h-4 w-4 my-auto ${
+                                                    className={`ml-2 h-4 w-4 my-auto ${
```

Comment:
**question**: Is `ml-1` not available?

```suggestion
                                                    className={`ml-1 h-4 w-4 my-auto ${
```

Reply:
Resolving this as the margin has been added on the other side. Cc @geropl

Reply:
Unresolving as this is about the sorting icon. Still relevant, @geropl! Minor.

Reply:
Jep, noticed as well. Ok to move into a follow up? Already missing this PR in prod. :upside_down_face:

Reply:
@geropl DEAL—Opened https://github.com/gitpod-io/gitpod/issues/12221 to keep track of this as a reminder and marked it as a good-first-issue in case anyone from the community would like to tackle this.